recipes/barrnap/0.3/build.sh

```diff
@@ -1,3 +1,3 @@
 #!/bin/sh
-mkdir -p $PREFIX
-mv ./* $PREFIX/
+mkdir -p $PREFIX/bin
+mv ./* $PREFIX/bin
```

Comment:
The usual way we handle this is to install everything into ``$PREFIX/share/<packagename>`` and either symlink the executables to $PREFIX/bin or create wrapper scripts (depending how robust the executables are written). There are a couple of examples out there in the repo.